articles/data-factory/connector-troubleshoot-guide.md

```diff
@@ -663,6 +663,27 @@ busy to handle requests, it returns an HTTP error 503.
 
 - **Recommendation**:  Check the sink dataset and fix the path without wildcard value.
 
+### Error code:  RemotePrivateLinkServiceSubscriptionNotRegistered
```

Comment:
Seems to me it shouldn't be under "General copy activity error" section, customer haven't reached copy activity yet.
From where can customer see this error message? Shall we move it to Data Factory UX/authoring or virtual network/security articles?